kinder/ci/tools/update-workflows/templates/workflows/instance-config-tasks.yaml

```diff
@@ -218,6 +218,36 @@ tasks:
         exit 1;
         }
 
+      exit 0
+  timeout: 5m
+- name: check-and-add-kubeadm-flags
```

Comment:
```suggestion
- name: check-and-add-flag
```

Reply:
fixed.

Reply:
i recommended `flag` not `flags`.
in follow up tasks you have `check-flag-after-second-upgrade`.

just try makaing the naming consistent.

Reply:
Thank you for your carefulness, fixed again.